js/bibox.js

```diff
@@ -1462,15 +1556,20 @@ module.exports = class bibox extends Exchange {
             this.checkRequiredCredentials ();
             if (version === 'v3' || version === 'v3.1') {
                 const timestamp = this.milliseconds ();
-                const strToSign = '' + timestamp + this.json (json_params); // 3. This is the string that needs to be signed
+                let strToSign = '' + timestamp;
+                if (json_params !== '{}') {
+                    strToSign += json_params;
+                }
                 const sign = this.hmac (this.encode (strToSign), this.encode (this.secret), 'md5').toString ();
                 headers['bibox-api-key'] = this.apiKey;
                 headers['bibox-api-sign'] = sign;
                 headers['bibox-timestamp'] = timestamp;
                 if (method === 'GET') {
                     url += '?' + this.urlencode (params);
                 } else {
-                    body = json_params;
+                    if (json_params !== '{}') {
+                        body = params;
```

Comment:
Is this intentionally `params` or should it be `json_params`?

Reply:
Yes, it fails when passing along `json_params`